### ISSUE TITLE
Smooth out some rough edges in UI

### ISSUE THREAD

**Issue by nokome** [stencila/stencila#154]
There are some relatively small, but annoying issues with the UI. These could distract from the core concept so it would be good to smooth them out soon.

- [x] Can't see all of the cell expression easily #145 
- [x] When you can't see all of the cell contents, it's not obvious that you can press the spacebar to toogle the display mode. Add a tool for cell display mode #143 
- [ ] When a cell is clipped, it would be nice to be temporarily overlay mode when mouse overed, click on it then and it stays overlay.
- [ ] Have to type in a cell id or cell range; can't use mouse #103

**Comment by michael** [comment]
Quick status update.

I made input fields quite big.

![image](https://cloud.githubusercontent.com/assets/2931/13729644/30ad98ec-e93a-11e5-9227-079c425f37b6.png)

Making them dynamically adapt is not so trivial. Either way we want syntax highlighting in the future (e.g. while you type an expression). So I think we should solve this all together and expect this will take some time to implement.

--------

Added a DisplayModeTool.

![image](https://cloud.githubusercontent.com/assets/2931/13729647/5f350b32-e93a-11e5-9056-9b8b7017f016.png)

I started an attempt to reach the mouseover functionality, but it's actually not so easy too. We'd need to kind of doublerender the contents so we can have one container showing the clipped version and another one rendering the expanded version in a popover. The markup will be quite polluted and I'm a bit afraid of side effects also (click handling etc). Would do this after the demo release as it's quite experimental.

--------

The reason why entering the commit message does not work is that we have a global key handler installed. In order to solve it we would need to temporary turn that off and back on once the commit message editing is done. We will end up with tricky side effects though if I just add a hack (like a window.disableGlobalKeyHandling variable). If you open the commit dialog and click back into the spreadsheet your key events would not be handled anymore. I want to avoid this hackish solution. Will talk with Oliver if we have better options.
